src/Administration/Resources/app/administration/test/factory/component.factory.spec.js

```diff
@@ -846,4 +846,76 @@ describe('core/factory/component.factory.js', () => {
         expect(typeof component.vm.$super).toBe('function');
         expect(component.vm.$super('fooBar')).toBe('fooBar');
     });
+
+    it('builds the super call chain if first component in inheritance chain does not implement overridden method', () => {
+        ComponentFactory.register('grand-mother-component', {
```

Comment:
I'd probably go with "grandparent-component" (not "grand-parent"!) as well as "parent-component" instead of the whole mother thing since that's a bit too quirky for my taste. Ditto for the other tests.